NeoMathEngine/src/CPU/x86/CpuX86MathEngineVectorMathPrivate.h

```diff
@@ -650,6 +650,11 @@ inline void vectorEltwiseMultiply( const int* first, const int* second, int* res
 inline void vectorEltwiseMultiplyAdd( const float* first, const float* second, float* result, int vectorSize )
 {
 #ifdef DEBUG_ENABLE_AVX2
+	if( vectorSize == 1 ) {
```

Comment:
Why this case under `#ifdef`?

Reply:
This define is just for the fast check before/after these changes.
It'll be removed in the end.